randomActivityGen.py

```diff
@@ -10,6 +11,9 @@
     -o, --output-file FILE      Write modified statistics to FILE
 
 Other Options:
+    --centre.pos args           The coordinates for the city's centre, e.g. "300,500" or "auto" [default: auto]
+    --centre.pop-weight F       The increase in population near the city center [default: 0.8]
```

Comment:
Intuitively I would think of 0.8 as being a 20% decrease, not an 80% increase, similar for centre.work-weight. But works as it is defined as an increase.

Reply:
Yeah, I see what you mean. But it is a weight for the "added" gradient, so it kinda works like an increase.